delfin/task_manager/scheduler/schedulers/telemetry/failed_telemetry_job.py

```diff
@@ -41,6 +41,22 @@ def __call__(self):
         :return:
         """
         try:
+            # Remove jobs from scheduler  for deleted failed_tasks
+            # Remove jobs from scheduler if it is added
+            filters = {'deleted': True}
+            failed_tasks = db.failed_task_get_all(self.ctx, filters=filters)
+            LOG.debug(" total failed_tasks found deleted "
```

Comment:
Please remove the duplicated space at the head and log should be started with upper case

Reply:
Fixed, Thanks